package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "adopt-an-orphan",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] Great job Team, however, I can not run your project locally, I first got an error message that says you miss the `npm err missing script start` but after adding a script start into your package.json file, I'm now getting this kind error message 👇🏻 Kindly fix this

![npm](https://user-images.githubusercontent.com/79058364/169873370-3c1a7aec-1680-4ca8-93b2-bd810536148b.png)

Reply:
Hi, @BenMukebo we just saw your suggestions and have applied them but we are a little confused in this comment is it something we have to incorporate in our project or you are going to set up it in your machine?

Reply:
There is nothing to incorporate guys, you just need to fix the error message which displays in the terminal when I'm running. You're running locally the project. 😄